concepts/staking.md

```diff
@@ -6,40 +6,26 @@ description: Details of Staking with Angle Protocol
 
-{% hint style="info" %}
-Staking contracts of the protocol may not be released immediately at launch of the protocol on mainnet.
-{% endhint %}
-
 ## 💐 Token Distribution
 
-There is one specific staking contract per sanToken and per agToken. The staking contracts for Hedging Agents are directly the contracts in which HAs' positions are stored.
-
-The fewer tokens are staked, the more interesting it is to participate in this staking contract because a similar amount of governance tokens are shared among a smaller amount of people.
+The distribution of governance tokens between the different staking contracts (called gauges) happens through weekly votes from veANGLE token holders. They assign specific weights to the different gauges, and the sum of their votes will dictate the next rewards distribution.
 
-The distribution of governance tokens to the different staking contracts is done by a contract controlled by governance. This contract stores for each staking contract a set of parameters, like a duration for the distribution (100 blocks, a day, a week, a month, a year, ...), and a number of tokens to distribute across this duration.
-
-400,000,000 ANGLE (40% of supply) are planned to be distributed through staking. The amount being distributed are divided by 1.5^(1/52) = 1.007827 every week, equivalent to dividing the distribution by 1.50 every year.
+400,000,000 ANGLE (40% of the total supply) is being distributed through staking. The amount being distributed are divided by 1.5^(1/52) = 1.007827 every week, equivalent to dividing the distribution by 1.50 every year.
 
 ![ANGLE Distribution](../.gitbook/assets/Liquidity-Mining-Distributed-Supply-Over-Time.png)
 
 We built some simulations to evaluate how supply evolves over time, you can take a look at this [Google Sheet](https://docs.google.com/spreadsheets/d/1yraSUH_7D-VMnCUsIYWWdW1pxL7bDxN3o0M5japQmeY/edit#gid=0).
 
 {% hint style="info" %}
-ANGLE token owners are able to participate to Snapshot votes each week to decide how to allocate the liquidity mining supply across different pools. This is detailed [here](../governance/voting.md)
+The voting process deciding the rewards allocation takes place each week, and is detailed [here](../governance/voting.md).
 {% endhint %}
-
-## 📶 Token Accumulation
-
-At each block, tokens are distributed based on the rate specified by governance. If 1000 governance tokens are to be distributed for in 100 blocks (e.g, 10 governance tokens distributed each block), then stakers receive at a each block a fraction of these 10 tokens depending on what they put in the staking contract and on what other stakers have put at the time of the block.
```

Comment:
I'd still keep this paragraph.
I'd also add a paragraph to say (here or somewhere else) that there are different staking contracts: on mainnet where veBoost applies and on other chains where it's a classical staking contract